packages/queue/middleware.js

```diff
@@ -6,6 +6,7 @@ import {
 } from '@bufferapp/async-data-fetch';
 import { actions as notificationActions } from '@bufferapp/notifications';
 import { actionTypes, actions } from './reducer';
+import {getURL} from "@bufferapp/publish-formatters";
```

Comment:
Small style thing, just spacing around the braces! 💅 

```suggestion
import { getURL } from "@bufferapp/publish-formatters";
```